### PR TITLE
Update Battery

### PR DESCRIPTION
**Pull request by bmclaughlin**
image

https://redhat.invisionapp.com/share/V4NY3C6XKRW#/screens/350113377
https://projects.engineering.redhat.com/browse/RHIADVISOR-471

### CODE REVIEW

**Comment by AllenBW** [comment]
@bmclaughlin can we get screen shots offfff this?

**Comment by bmclaughlin** [comment]
Working on screenshots.

**Comment by ryelo** [comment]
@bmclaughlin Can we get an example of these in one of the tables? Just a screenshot.

**Comment by AllenBW** [comment]
Does this need ux input/review?

**Comment by bmclaughlin** [comment]
image

File: src/PresentationalComponents/Battery/CriticalBattery.js [hunk 1]
```diff
@@ -1,11 +1,14 @@
...
 import React from 'react';
 
 const CriticalBattery = () => {
     return (
-        <path d="M16,212h416c8.8,0,16-7.2,16-16v-40c0-8.8-7.2-16-16-16H16c-8.8,0-16,7.2-16,16v40C0,204.8,7.2,212,16,212z M16,372h416
-        c8.8,0,16-7.2,16-16v-40c0-8.8-7.2-16-16-16H16c-8.8,0-16,7.2-16,16v40C0,364.8,7.2,372,16,372z M16,512h416c8.8,0,16-7.2,16-16v-40
-        c0-8.8-7.2-16-16-16H16c-8.8,0-16,7.2-16,16v40C0,504.8,7.2,512,16,512z M16,72h416c8.8,0,16-7.2,16-16V16c0-8.8-7.2-16-16-16H16
-        C7.2,0,0,7.2,0,16v40C0,64.8,7.2,72,16,72z"/>
+        <React.Fragment>
+            <path d="M 99.168858,143.38516 H 351.33914 c 5.33437,0 9.69886,-5.04 9.69886,-11.2 v -28 c 0,-6.16 -4.36449,-11.2 -9.69886,-11.2 H 99.168857 c -5.334371,0 -9.698857,5.04 -9.698857,11.2 v 28 c 0,6.16 4.364486,11.2 9.698858,11.2 z" />
```
[the remark this answers is not in the document]
Comment (bmclaughlin) [reply src/PresentationalComponents/Battery/CriticalBattery.js]: Didn't realize you could group paths, thanks!

**Comment by bmclaughlin** [comment]
@ryelo, table screenshot and grouped paths (:tada:) updated.

**Comment by ryelo** [comment]
@bmclaughlin You may want to increase the size of them

**Comment by bmclaughlin** [comment]
@ryelo, updated the screenshot above, and thanks!

**Comment by ryelo** [comment]
@bmclaughlin LGTM

cc: @katierik

**Pull request merged**